app/src/app/[lng]/data/review/page.tsx

```diff
@@ -92,6 +113,14 @@ export default function ReviewPage({
       router.push("/");
       dispatch(clear());
       setIsConfirming(false);
+
+      sendEmailNotification({
+        cityName: cityData?.name,
+        files: uploadFilesDetail,
+      }).then((res) => {
+        //  TODO
+        // trigger notification to user
+      });
```

Comment:
Should not happen here. Also no user notification necessary, it just happens when the file is done uploading and needs no user feedback.